easy_run.py

```diff
@@ -40,6 +41,24 @@ def main():
     transfer_assignments_to_todoist()
     canvas_assignment_stats()
     print("Done!")
+    background()
+
+
+def background():
+    """After setup is finished, it asks the user if they want to have it run again in the background.
+    """
+    global background_activated
+    if background_activated:
+        # Don't run again if it's already running in the background
+        return
+    
+    background_activated = yes_no("Leave running in the background?")
+    
+    if background_activated:
+        while True:
+            print("Time to sleep...")
+            time.sleep(600)
```

Comment:
Can the sleep time be moved to another top variable config for easy finding/change for other users?

Reply:
Sure thing. Thanks!

Reply:
Fixed in cb62e1ab17f7470d1d5258f680f6183d03c39ead